app/models/patient_import_row.rb

```diff
@@ -89,6 +89,13 @@ def to_school_move(patient)
         end
 
       school_move.tap { _1.source = school_move_source }
+
+      if patient.organisation.nil?
```

Comment:
I don't think we should be calling this here because they're handled in bulk for performance reasons.

I think we need to add the logic to here instead to say if the organisation is nil then the school move needs to be confirmed: https://github.com/nhsuk/manage-vaccinations-in-schools/blob/17d220839ae5891d62fcca4423769346996f10e3/app/models/patient_import.rb#L35